astropy/coordinates/transformations/base.py

```diff
@@ -0,0 +1,115 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+"""Coordinate transformations base class.
+
+This module contains the abstract base class for coordinate transformations. For
+concrete implementations, see the submodules
+:mod:`~astropy.coordinates.transformations.affine` and
+:mod:`~astropy.coordinates.transformations.function`. A sequence of transformations can
+be combined into a single transformation using
+:class:`~astropy.coordinates.transformations.CompositeTransform`.
```

Comment:
I'd tend to just give the `composite` module here, since this is aimed a bit more at developers (and users should import `astropy.coordinates.CompositeTransform`).

Reply:
✅